src/game.cpp

```diff
@@ -704,6 +716,38 @@ class GameGlobalShaderConstantSetter : public IShaderConstantSetter
 			sunlight.b };
 		m_day_light.set(dnc, services);
 
+		// Lighting will also use location-based lighting in the future
+		// for smaller models so the sun color alone isn't sufficient
+		const float time_of_day = (float)m_client->getEnv().getTimeOfDayF();
+		float sun_angle = time_of_day * M_PI * 2;
+		float light_color[4] = {
+			sunlight.r,
+			sunlight.g,
+			sunlight.b,
+ 			1.0 };
+		light_color[3] = 1.0 - (time_of_day / 1000.0);
```

Comment:
Should the line a bit below that (`sun_angle -= M_PI * (1.0 - ((daynight_ratio - 320) / 150.0))`) also use `150.0f`?